src/Components/DrillAnimation.js

```diff
@@ -1,23 +1,24 @@
 import React, { useState } from 'react';
 import { ScrollView, StyleSheet, View, Text, Dimensions, TouchableOpacity } from 'react-native';
 import { WebView } from 'react-native-webview';
-import { LinearGradient } from 'expo-linear-gradient';
 
 import Animation from './Animation';
 import VimeoVideo from './VimeoVideo';
+import GradientButton from './shared/GradientButton';
+import { Sources } from '../Fixtures';
 import theme from '../styles/theme.style';
 
 const DrillAnimation = props => {
```

Comment:
Following my question on the `SourceType` enum, I think the same questions pops out here : We use the word Animation for two purposes and it makes it more difficult to follow

DrillAnimation : Animation or video
Animation : Animated diagram

Whe should go with different words for the two concepts

Suggestion :
- DrillIllustration : animation or vidéo
Animation : animated Diagram
- DrillAnimation : Diagram or video
Diagram : animated diagram

What do you think?

Reply:
I agree with you. I would remove the word Animation as it could be anything.
So, I would choose 
- `DrillIllustration` : animation or video
- `Diagram`: animated diagram

Is it ok for you ?

Reply:
Yep

Reply:
Maybe we should ask on Slack if everyone is ok with a renaming Animation->Diagram, since it will cause conflicts ?

Reply:
Sure. And I don't think the best option is to do it but better with the Zach's PR